### PR TITLE
Cleanup iframes when running tests on chrome.

### PR DESCRIPTION
**Pull request by mkhatib**
Removes test iframes to address some of the issues with memory leaks #3138

### CODE REVIEW

**Comment by mkhatib** [comment]
Filed #3315 and #3314 to follow up with the issues in Safari and the skipped test.

File: test/_init_tests.js [hunk 1]
```diff
@@ -76,7 +77,12 @@ sinon.sandbox.create = function(config) {
 // Global cleanup of tags added during tests. Cool to add more
 // to selector.
 afterEach(() => {
-  const cleanup = document.querySelectorAll('link,meta');
+  const cleanupTagNames = ['link', 'meta'];
+  if (platform.isChrome()) {
```
Comment (cramforce) [inline test/_init_tests.js]: `!platform.isSafari()`

File: test/functional/test-iframe-helper.js [hunk 2]
```diff
@@ -99,7 +99,9 @@ describe('iframe-helper', function() {
     });
   });
 
-  it('should un-listen on next message when iframe is unattached', () => {
+  // TODO(mkhatib): Figure out why this fails. Probably have to do with removing
```
Comment (cramforce) [inline test/functional/test-iframe-helper.js]: Put issue number instead of your name :)

**Comment by cramforce** [comment]
Does this pass tests in Safari?

File: test/_init_tests.js (hunk 1 above)
Comment (mkhatib) [inline test/_init_tests.js]: Done

File: test/functional/test-iframe-helper.js (hunk 2 above)
Comment (mkhatib) [inline test/functional/test-iframe-helper.js]: Done.

File: test/_init_tests.js (hunk 1 above)
Comment (cramforce) [inline test/_init_tests.js]: Not pushed yet.

**Comment by cramforce** [comment]
LGTM

File: test/_init_tests.js (hunk 1 above)
Comment (mkhatib) [inline test/_init_tests.js]: It was, Github sometimes doesn't seem to update the diff view for some reason. Now updated.

**Pull request merged**